### PR TITLE
homeserver input field

### PR DESCRIPTION
**Pull request by f0x52**
I guess this would work? I just edited it in vim idk android stuff

### CODE REVIEW

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -43,6 +43,16 @@
                 android:inputType="textPassword"
                 android:maxLines="1" />
 
+            <EditText
+                android:id="@+id/hsField"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:ems="10"
+                android:hint="Homeserver"
```
Comment (krombel) [inline app/src/main/res/layout/activity_login.xml]: Would it be possible to directly use strings here? (especially useful for translation)

Comment (f0x52) [reply app/src/main/res/layout/activity_login.xml]: In which way? I just copied how the username and password fields work

Comment (krombel) [reply app/src/main/res/layout/activity_login.xml]: Then a separate PR for i10n has to be created either way. Just saw the diff and not what was done for the other input fields

Comment (ganfra) [reply app/src/main/res/layout/activity_login.xml]: @krombel:  Yes, at the moment, some strings are still hard-coded, and should be cleaned. 
I'll probably do that this week.

Comment (ganfra) [reply app/src/main/res/layout/activity_login.xml]: @f0x52 : could you please reproduce the behavior actual Riot has on this screen with a checkbox to show/hide additional auth settings, including homeserver, with default to matrix server? 
Thanks for your work!

Comment (jcgruenhage) [reply app/src/main/res/layout/activity_login.xml]: @ganfra I think this should also be merged as is, and the behaviour Riot has can be implemented in a later PR. This PR is already immensely useful for testers as it is.

Comment (jcgruenhage) [reply app/src/main/res/layout/activity_login.xml]: (Under the assumption that it works, of course)

Comment (f0x52) [reply app/src/main/res/layout/activity_login.xml]: @ganfra I quickly made this to speed up easier testing, I don't really know android development so I probably can't help with more complicated behaviour

Comment (ganfra) [reply app/src/main/res/layout/activity_login.xml]: No problem, I just made it quickly on my side. Thanks for your work anyway :)

**Pull request closed without merging**